RiotSharp/StaticDataEndpoint/StatsStatic.cs

```diff
@@ -75,100 +75,100 @@ public class StatsStatic
 
         public double PercentSpellVampMod { get; set; }
 
-        [JsonProperty("rFlatArmorModPerLevel")]
-        public double RFlatArmorModPerLevel { get; set; }
+        [JsonProperty("FlatArmorModPerLevel")]
+        public double FlatArmorModPerLevel { get; set; }
```

Comment:
The property name seems to match `JsonProperty`, can't we get rid of the `JsonProperty`?